### PR TITLE
Add server SSL debug sectn; edit debug for Presto

### PR DESCRIPTION
**Pull request by Ordinant**
On request, added missing info on logging.properties entry to aid SSL debugging.

Plus minor English cleanup of the next section, SSL debugging for the CLI.

### CODE REVIEW

File: presto-docs/src/main/sphinx/security/ldap.rst [hunk 1]
```diff
@@ -310,11 +310,23 @@ Java Keystore File Verification
 Verify the password for a keystore file and view its contents using
 :ref:`troubleshooting_keystore`.
 
+SSL debugging for the server
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/security/ldap.rst:313]: Title case should be same as in rest of doc. Overall we should change all docs in prestosql but that has to be done in one sweep.

**Review by mosabua — APPROVED** [review]
One small change requested but overall good to go.

File: presto-docs/src/main/sphinx/security/ldap.rst [hunk 2]
```diff
@@ -310,11 +310,23 @@ Java Keystore File Verification
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To add debugging log messages from the Presto server about TLS/SSL connection
+errors, add the following line to the ``logging.properties`` file in the
+server's ``etc`` directory:
+
+.. code-block:: text
+
+    io.prestosql.plugin.password=DEBUG
```
Comment (electrum) [inline presto-docs/src/main/sphinx/security/ldap.rst:322]: The title is "SSL debugging for the server", but this is actually about debugging issues with the Presto server talking to the LDAP server. SSL errors are just one type of thing you might need to debug.

Comment (Ordinant) [reply presto-docs/src/main/sphinx/security/ldap.rst:322]: (The requesting party asked to add instructions on SSL debugging.)

Let's see, how about instead "Debug Presto to LDAP Server Issues"

Comment (Ordinant) [reply presto-docs/src/main/sphinx/security/ldap.rst:322]: New commit pushed.

Comment (electrum) [reply presto-docs/src/main/sphinx/security/ldap.rst:322]: That sounds good. Let's remove/generify the TLS/SSL part.

I also dislike repeating the part about `logging.properties`. How about we make that a link

```
If you need to debug issues with Presto communicating with the LDAP server,
you can change the :ref:`log level <log-levels>` for the LDAP authenticator:
```
You'll need to add an anchor above the "Log Levels" section in `deployment.rst`:
```
.. _log_levels:

Log Levels
^^^^^^^^^^
```

File: presto-docs/src/main/sphinx/security/ldap.rst [hunk 3]
```diff
@@ -310,11 +310,23 @@ Java Keystore File Verification
...
 :ref:`troubleshooting_keystore`.
 
+SSL debugging for the server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To add debugging log messages from the Presto server about TLS/SSL connection
+errors, add the following line to the ``logging.properties`` file in the
+server's ``etc`` directory:
+
+.. code-block:: text
```
Comment (electrum) [inline presto-docs/src/main/sphinx/security/ldap.rst:320]: Use `none` to be consistent

File: presto-docs/src/main/sphinx/security/ldap.rst [hunk 4]
```diff
@@ -310,11 +310,23 @@ Java Keystore File Verification
...
+    io.prestosql.plugin.password=DEBUG
+
 SSL Debugging for Presto CLI
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you encounter any SSL related errors when running Presto CLI, you can run CLI using ``-Djavax.net.debug=ssl``
-parameter for debugging. You should use the Presto CLI executable jar to enable this. E.g.:
+If you encounter any SSL related errors when running the Presto CLI, you can run
+the CLI using the ``-Djavax.net.debug=ssl`` parameter for debugging. Use the
+Presto CLI executable Jar to enable this. For example:
```
Comment (electrum) [inline presto-docs/src/main/sphinx/security/ldap.rst:329]: JAR should be uppercase

File: presto-docs/src/main/sphinx/security/ldap.rst (hunk 2 above)
Comment (Ordinant) [reply presto-docs/src/main/sphinx/security/ldap.rst:322]: Coming up.

Comment (Ordinant) [reply presto-docs/src/main/sphinx/security/ldap.rst:320]: Et voila.

File: presto-docs/src/main/sphinx/security/ldap.rst (hunk 4 above)
Comment (Ordinant) [reply presto-docs/src/main/sphinx/security/ldap.rst]: Et volia, redux.

**Review by electrum — APPROVED** [review]

**Pull request merged**